music/src/core/player.ts

```diff
@@ -201,7 +210,7 @@ export abstract class BasePlayer {
    * Stop playing the currently playing sequence right away.
    */
   stop() {
-    if (this.currentPart) {
+    if (this.isPlaying()) {
```

Comment:
I don't think this is correct. The player could be paused, and you should still be allowed to stop it.

Reply:
`!!this.currentPart` is the definition of `isPlaying()`, so this behaves the same way as before. I just thought it was clearer to write it like this.

Reply:
ah, good call.